themes/stanford_basic/decoupled-menu/src/main-menu.island.tsx

```diff
@@ -298,6 +313,10 @@ const ListItem = styled.li<{ level?: number }>`
   @media (min-width: 992px) {
     border-bottom: ${props => props.level === 0 ? "none" : "1px solid #d9d9d9"};
   }
+  
+  @media (max-width: 992px) {
+    padding: ${props => props.level > 0 ? "0 0 0 10px" : "0"};
```

Comment:
Move this outside of the media query. just edit line 306, then you can do padding: 0 inside the min-width query.